data/hai/hai reveal 2 start.txt

```diff
@@ -0,0 +1,368 @@
+	npc
+		government "Bounty Hunter that Won't Enter Hai Space"
+		personality plunders disables unconstrained
+		fleet "Large Core Pirates" 1
+		fleet "Small Core Pirates" 1
+			
+
+mission "Hai Rescue: Gipeep"
+	landing
+	name "Return Gipeep"
+	description "Transport the Hai you rescued to <destination>. Payment is <payment>."
+	source "Maelstrom"
+	destination "Greenbloom"
+	passengers 1
+	to offer
+		"combat rating" > 50
+		not "event: hai-human treaty signed"
+		has "Unwanted Cargo: done"
+		has "Hai Leaks Response 6: active"
+		# Some subset of these are restricted so you can't pick up enough of them to finish the quest at the same time as starting it.
+	on offer
+		conversation
+			`You walk by some cargo and hear loud thumping coming from a locked crate. You notice a key nearby. Do you want to check it out?`
+			choice
+				`	(Examine the crate.)`
+				`	(Leave.)`
+					decline
+			`	The key has a symbol you don't recognize, but it looks like an angry cat in a circle. As you get closer, the ordinary-looking crate starts seeming a little strange. It appears to be well-insulated against sound and temperature, but has concealed breathing holes. You can still hear loud thumping.`
+			`	Do you want to try opening the crate?`
+			choice
+				`	(Open the crate.)`
+				`	(Leave.)`
+					decline
+			`	You put the key in the lock, turn, and the next thump you hear pushes the crate open so hard it falls over. An angry-looking Hai jumps out and starts yelling at you in its own language.`
+			`	After a few seconds, the Hai calms down and speaks in your own tongue, "They locked me in a box! I was walking home from a store on <destination> and some humans pulled me into an alley. They shoved me in a box and now I'm..." the Hai pauses and looks around, "not in Hai space?"`
+			`	"Please take me home. I'm a ship architect from <destination>. We should hurry before they come back."`
+			choice
+				`	"Of course. Just hop in my ship."`
+					goto accept
+				`	"No, I'm not going that way any time soon."`
+			`	"I have nowhere else to go! I'm not exactly conspicuous. I'll be caught soon. Please, I can pay you well."`
+			choice
+				`	"In that case, I'll take you."`
+				`	"No. You'll have to find another pilot."`
+					decline
+			label accept
+			`	You can hear men walking down the hall towards you. You and Gipeep run quickly in the other direction. Soon you hear shots fired, and see them hit a nearby wall. They passed quite close to your head.`
+			`	Through luck and skill, the two of you make it to your ship and quickly launch. The escape seemed a little too easy though; they may have been avoiding hitting Gipeep.`
+			`	It appears ships are following you into orbit.`
+				launch
+	on complete
+		log `Rescued a Hai ship architect named Gipeep who was kidnapped near his home on <destination> and transported in a locked cargo crate. The crate's key had a marking on it similar to the Syndicate logo.`
```

Comment:
Maybe the player should note when earlier investigating the key that it looks like the Syndicate logo?

Reply:
This is an oversight from an earlier version of the story, the symbol in fact does *not* look like the Syndicate logo.